### ISSUE TITLE
Option to resubmit failed requests

### ISSUE THREAD

**Issue by jonasjucker** [C2SM/zonda-request#78]

**Comment by mjaehn** [comment]
And additionally, they can always re-submit manually by just submitting a new issue with the same JSON.

**Comment by stelliom** [comment]
Yes, that too. I guess the *issue-\** branches are getting cleaned up somehow, right?

**Comment by mjaehn** [comment]
@stelliom Imho this can be closed as you have implemented a solution in Jenkins that enables re-running and debugging failed requests.
